x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.tsx

```diff
@@ -26,86 +29,452 @@ interface AnalyticsCollectionIntegrateProps {
   collection: AnalyticsCollection;
 }
 
+type TabKey = 'javascriptEmbed' | 'searchuiEmbed' | 'javascriptClientEmbed';
+
 export const AnalyticsCollectionIntegrate: React.FC<AnalyticsCollectionIntegrateProps> = ({
```

Comment:
nitpicking now but this file is really, really long. Might be worth considering splitting it up into a few different files/components.

Reply:
Yeh! Was borderline but likely to grow aswell. I've split it up for future changes.